docs/user/rstudio/ide/guide/troubleshooting/resetting-state.qmd

```diff
@@ -0,0 +1,98 @@
+---
+title: "Resetting RStudio's State"
+date-meta: 2025-01-08
+---
+
+# Overview
+
+RStudio Desktop stores its internal state in a hidden directory. If this directory does not exist, RStudio will create it on start up. This directory includes information about open documents, log files, and other state information. Removing (or renaming) this directory will reset RStudio's state. 
+
+We recommend renaming this directory to create a backup version instead of completely deleting it. If you experience a crash or RStudio fails to start, this directory may contain vital information for determining the source of the error. In this case, we recommend renaming this directory and sending it along if asked by Posit's support team.
```

Comment:
```suggestion
Instead of deleting this directory, we recommend renaming this directory to create a backup version. If you experience a crash or RStudio fails to start, this directory contains vital information for determining the source of the error. In this case, we recommend renaming this directory and forwarding it to Posit's support team.
```